server/models/handlers/appointment.js

```diff
@@ -0,0 +1,16 @@
+const appointmentModel = require('../db/appointment');
+
+// Adds an appointment to the database and returns true upon successful completion.
+async function bookAppointment(studentId, workerTimeslotId, purpose, studentNotes, workerComments) {
+    try {
+        await appointmentModel.insertAppointment(studentId, workerTimeslotId, purpose, studentNotes, workerComments);
+        return true;
```

Comment:
Currently, returning a boolean value that indicates successful insertion of the appointment. Is that acceptable or would you prefer if the entire appointment object be returned instead?

Reply:
I think it is okay to leave it as currently implemented. It would make sense for true to be returned for the successful insertion of the appointment and false to indicate an error that occurred.

Reply:
Sounds good, thank you for the comment!